src/java.desktop/windows/native/libawt/windows/awt_DnDDT.cpp

```diff
@@ -225,6 +225,7 @@ HRESULT __stdcall AwtDropTarget::DragEnter(IDataObject __RPC_FAR *pDataObj, DWOR
 
 HRESULT __stdcall AwtDropTarget::DragOver(DWORD grfKeyState, POINTL pt, DWORD __RPC_FAR *pdwEffect) {
     TRY;
+    AwtToolkit::GetInstance().isInDoDragDropLoop = TRUE;
```

Comment:
This is a new addition. Did you miss this function in previous iteration?

Reply:
Good catch! Actually that line is harmless but it is really unnecessary. I will update PR